nifi-nar-bundles/nifi-aws-bundle/nifi-aws-processors/src/main/java/org/apache/nifi/processors/aws/s3/PutS3Object.java

```diff
@@ -407,6 +420,12 @@ public void process(final InputStream rawIn) throws IOException {
                             }
                         }
 
+                        final String Sse = context.getProperty(SERVER_SIDE_ENCRYPTION).getValue();
```

Comment:
Minor note -- naming convention would be `sse`, or preferably, `serverSideEncryption`.